### PR TITLE
Update report template and the corresponding tryton Report for Vara Report.

### PR DESCRIPTION
**Pull request by narendraj9**
Asana tasks: 
https://app.asana.com/0/1204656475814831/1204666080503886/f
https://app.asana.com/0/1204656475814831/1204666080503908/f

Changes in this PR: 
- Update report template for the report "Vara Report" and the corresponding Tryton Report class as specified in the respective Asana task.
- Override the default value for "Name Representation" of a Party to be in the format "\<Given Name\> \<Family Name\>"

### CODE REVIEW

File: view/mammography_patient_form.xml [hunk 1]
```diff
@@ -40,9 +40,17 @@ contains the full copyright notices and license terms. -->
     <group id="patient_biopsies">
         <field name="biopsies" string="" colspan="4"/>
     </group>
-    <separator string="Other important information" id="sep_patient_other_info" colspan="4"/>
+    <separator name="general_info" id="sep_patient_other_info" colspan="4"/>
```
Comment (DanielStephens) [inline view/mammography_patient_form.xml:43]: was this changed to observations by the translation level. DO we need to be aware to set up some translation files during VPP deployment?

File: report/patient_vara_findings.fodt [hunk 2]
```diff
@@ -1,31 +1,40 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (DanielStephens) [inline report/patient_vara_findings.fodt:1]: What generates this file?

File: report/patient_vara_findings.fodt [hunk 3]
```diff
@@ -147,35 +148,37 @@
   </office:script>
  </office:scripts>
  <office:font-face-decls>
-  <style:font-face style:name="Andale Sans UI" svg:font-family="&apos;Andale Sans UI&apos;" style:font-family-generic="system" style:font-pitch="variable"/>
-  <style:font-face style:name="Arial" svg:font-family="Arial"/>
-  <style:font-face style:name="Arial1" svg:font-family="Arial" style:font-family-generic="swiss"/>
-  <style:font-face style:name="DejaVu Sans" svg:font-family="&apos;DejaVu Sans&apos;" style:font-family-generic="system" style:font-pitch="variable"/>
+  <style:font-face style:name="Arial1" svg:font-family="Arial"/>
```
Comment (atamis) [inline report/patient_vara_findings.fodt:151]: Was this font change intentional?

File: health_vara_report.py [hunk 4]
```diff
@@ -21,69 +34,64 @@ def execute(cls, ids, data):
...
+            imaging_result = imaging_results[0]
+            findings = findings + list(imaging_result.findings)
+            density = imaging_result.density
+            assessment = imaging_result.assessment
+
+        # Processing findings outside the imaging results workflow.
+        imaging_finding_ids = set(map(lambda f: f.id, findings))
+        other_findings = list(filter(lambda f: f.id not in imaging_finding_ids,
+                                     patient.findings))
+        findings = findings + other_findings
```
Comment (DanielStephens) [inline health_vara_report.py:69]: At this stage, do we have any guarantee that we have at least one finding?
I can't track it down but maybe it's a check from before this method?

We seem to be relying on it a bit when finding `last_finding` density on line 76.

File: report/patient_vara_findings.fodt (hunk 3 above)
Comment (atamis) [reply report/patient_vara_findings.fodt:151]: There are actually a lot of font changes, but a lot of them are from `Arial1` to `Arial`, so it seems like Arial is the intended font.

Comment (narendraj9) [reply report/patient_vara_findings.fodt:151]: The template was changed according to how Hannes wanted it to look like. These changes are done by LibreOffice and not directly on the text file.

File: health_vara.py [hunk 5]
```diff
@@ -29,6 +29,10 @@ def view_attributes(cls):
     def default_gender():
         return 'f'
 
+    @staticmethod
+    def default_name_representation():
+        return 'gf'
```
Comment (atamis) [inline health_vara.py:41]: Is there any documentation we can link to to better explain this?

File: view/mammography_patient_form.xml (hunk 1 above)
Comment (narendraj9) [reply view/mammography_patient_form.xml:43]: In this PR, I have only made this translatable. I am going to change the translation for this via the UI on each instance. Later, we can think about overriding the default translation for it in the module itself. I didn't have time to explore how to do it this time.

File: report/patient_vara_findings.fodt (hunk 2 above)
Comment (narendraj9) [reply report/patient_vara_findings.fodt:1]: It's generated with https://www.libreoffice.org/ and the templating is done with python-relatorio: https://relatorio.readthedocs.io/en/latest/

File: report/patient_vara_findings.fodt (hunk 3 above)
Comment (narendraj9) [reply report/patient_vara_findings.fodt:151]: Yes, this was intentional.

File: health_vara.py (hunk 5 above)
Comment (narendraj9) [reply health_vara.py:41]: I've added a comment about the allowed values.

File: health_vara_report.py (hunk 4 above)
Comment (narendraj9) [reply health_vara_report.py:69]: > At this stage, do we have any guarantee that we have at least one finding?

It's possible for the user to use this report with no findings. In that case, most of the values provided by the context are going to be `None`. We use `last_finding` in this method to get the density only if there is at least one item in `other_findings`.

Comment (DanielStephens) [reply health_vara_report.py:69]: Ahh okay, empty lists in python are falsy, I forgot, all good!

**Review by DanielStephens — APPROVED** [review]

File: view/mammography_patient_form.xml (hunk 1 above)
Comment (narendraj9) [reply view/mammography_patient_form.xml:43]: I've added a translation for this field.

**Review by atamis — APPROVED** [review]

**Pull request merged**